tests/hpc/slurm_slave.pm

```diff
@@ -12,7 +12,7 @@
 # Maintainer: soulofdestiny <mgriessmeier@suse.com>
 # Tags: https://fate.suse.com/316379, https://progress.opensuse.org/issues/20308
 
-use base "hpcbase";
+use base "opensusebasetest";
```

Comment:
I think that munge start, which is few lines bellow should updated.

Reply:
Changed!